conf.default.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I think we have this info in env.dist -- did you meant to commit this?

https://github.com/mozillascience/PaperBadger/blob/master/env.dist